zerver/lib/test_classes.py

```diff
@@ -587,6 +587,8 @@ def submit_reg_form_for_user(
             'default_stream_group': default_stream_groups,
             'source_realm': source_realm,
         }
+        if source_realm == '':
```

Comment:
Can't the default value of `source_realm` be set to "on" instead in submit_reg_form_for_user function signature?